src/Servers/IIS/AspNetCoreModuleV2/OutOfProcessRequestHandler/serverprocess.cpp

```diff
@@ -1866,13 +1879,16 @@ SERVER_PROCESS::~SERVER_PROCESS()
         m_pProcessManager = NULL;
     }
 
-    if (m_hStdoutHandle != NULL)
+    if (m_hStdErrWritePipe != NULL)
     {
-        if (m_hStdoutHandle != INVALID_HANDLE_VALUE)
+        if (m_hStdErrWritePipe != INVALID_HANDLE_VALUE)
         {
-            CloseHandle(m_hStdoutHandle);
+            THROW_LAST_ERROR_IF(!FlushFileBuffers(m_hStdErrWritePipe));
```

Comment:
Should not throw in destructor

Reply:
Yeah, we should really ignore the return of FlushFileBuffers. We don't care if it fails.